tests/get_or_create/tests.py

```diff
@@ -238,7 +240,11 @@ def test_get_or_create_raises_IntegrityError_plus_traceback(self):
             ManualPrimaryKeyTest.objects.get_or_create(id=1, data="Different")
         except IntegrityError:
             formatted_traceback = traceback.format_exc()
-            self.assertIn("obj.save", formatted_traceback)
+
+            # Reading from the code object allows this to work in pyc-only environments:
+            filename = re.escape(Model.save.__code__.co_filename)
+
+            self.assertRegex(formatted_traceback, f"{filename}.*save\n")
```

Comment:
This test was added in 31b1cbc623c246570e7301c0334df938d840638f, but is no longer needed, as the fix was reverted in 746caf3ef821dbf7588797cb2600fa81b9df9d1d without any consequences, so it now tests Python behavior rather than Django.

#17895